src/api.js

```diff
@@ -2,7 +2,7 @@ import axios from "axios";
 
 export const getConferenceList = async () => {
   try {
-    const response = await axios.get("localhost:8080/conferences");
+    const response = await axios.get("http://localhost:8080/conferences");
```

Comment:
For now it's probably alright to keep it there. In the future you probably have some config file with those urls :)